perf/benchmark_generic_mtrie.cpp

```diff
@@ -0,0 +1,141 @@
+/*
+    Copyright (c) 2023 Contributors as noted in the AUTHORS file
+
+    This file is part of libzmq, the ZeroMQ core engine in C++.
+
+    libzmq is free software; you can redistribute it and/or modify it under
```

Comment:
The license has changed, please check other files and update the header to match